docs/guides/opera_rtc_product_guide.md

```diff
@@ -0,0 +1,512 @@
+    select only the RTC-STATIC product type to return only the static layers that correspond to that RTC footprint.
+
+### Duplicate layer names
+
+There are **Local Incidence Angle** and **Mask** files listed in association with both the `RTC` search results 
+and the `RTC-STATIC` search results. 
+
+#### Local Incidence Angle
+
+The Local Incidence Angle file can be downloaded using either link, but it is the same file. It is always named with 
+this pattern: 
+`OPERA_L2_RTC-S1-STATIC_Txxx-xxxxxx-IWx_20140403_S1A_30_v1.0_local_incidence_angle.tif`
+
+The download URL behind the local incidence angle listings in both the `RTC` and `RTC-STATIC` results 
+reference the same source file.
+
+#### Validity Mask
+
+There is a file called `Mask` listed with both the `RTC` and the `RTC-STATIC` search results. Unlike the 
+local incidence angle file, these two mask files are NOT the same. Both are validity masks with the same pixel 
+value categories: 
+
+| Pixel Value | Description                                    |
+|-------------|------------------------------------------------|
+| 0           | Valid sample not affected by layover or shadow |
+| 1           | Valid sample affected by shadow                |
+| 2           | Valid sample affected by layover               |
+| 3           | Valid sample affected by layover and shadow    |
+| 255         | Invalid sample (fill value)                    |
+
+The main difference between the two files is that the data extent matches the other associated files. The 
+`RTC-STATIC` files have a larger extent than the `RTC` files, as data is included for the full raster footprint, 
+including the NoData pixels that are present around the edges of the RTC data products.
+
+![Figure 5](../images/opera-mask-compare.png "Comparison of the two OPERA RTC-S1 validity masks")
+
+The mask file in the map on the left is the mask linked to the `RTC` search results.
+
+- `OPERA_L2_RTC-S1_T115-245714-IW1_20250418T141628Z_20250419T010229Z_S1A_30_v1.0_mask.tif`
+
+The extent of NoData padding around area that has valid radar data for that particular burst is displayed with a 
+transparent pink color in this illustration, but would normally appear transparent. The mask values are only applied 
+to the pixels with valid radiometry within the radar burst.
+
+The mask file in the map on the right is the mask linked to the `RTC-STATIC` search results.
+
+- `OPERA_L2_RTC-S1-STATIC_T115-245714-IW1_20140403_S1A_30_v1.0_mask.tif`
+
+It includes validity mask values for the full extent of the burst footprint, including the NoData padding around the 
+pixels with valid radiometry within the radar burst. All of the static layers include data for this entire area.
+
+There may, however, also be differences in the actual pixel values when comparing an RTC-STATIC validity mask to 
+the validity mask included with a specific RTC product. Slight differences in the orbit with each acquisition means 
```

Comment:
```suggestion
the validity mask included with a specific RTC product. Slight differences in the orbit with each acquisition mean 
```

Reply:
In looking at this sentence, I ended up restructuring it for (hopefully) more clarity.